airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -40,59 +40,88 @@
 
 logger = logging.getLogger("airbyte")
 
+# deprecated from ConnectorSpec, using default value of 1 = 1 day.
+# Related Issue here: https://github.com/airbytehq/airbyte/issues/8027
+INSIGHTS_DAYS_PER_JOB: int = 7
```

Comment:
why it says 1 in comment and has value of 7?

Reply:
Here is why: https://github.com/airbytehq/airbyte/pull/8234#issuecomment-979488000

Reply:
Oh, my bad. Corrected.

> 

@keu What are the root causes of the possible job failure with date_slices? I've tested using the values of 300 and 1000 for the `insights days per job` - all are successfully processed on our test account. 
Therefore we can try to stay on value of 3 or 5 as a default value. WDYT?